### ISSUE TITLE
create User Groups page

### ISSUE THREAD

**Issue by agarwal** [ocaml/v2.ocaml.org#97]
We should list user groups, e.g. NYU OCaml Meetup and OCaml Users in PariS (OUPS), on a new page, and link to it from the Community menu. These are awkwardly linked to from the Meetings page currently, but it doesn't make sense there.

**Comment by Chris00** [comment]
OK but the page will look rather empty!  We could add some pictures but I am not sure it will make a big difference.  Maybe one should rename the page "User groups and meetings"?

There used to be regular meetings in Japan and Australia.  Not sure whether the groups still exist though.

**Comment by avsm** [comment]
Cambridge has one too; /cc @mcclurmc for more info.

**Comment by agarwal** [comment]
True, there are just about 5 groups currently. If each has a logo, it might
not look too sparse. But I'm okay with User Groups being a section on top
of the Meetings page too.

**Comment by amirmc** [comment]
Relevant to #113
